pilot/control/payloads/eventservice.py

```diff
@@ -229,38 +32,47 @@ def run_payload(self, job, out, err):
         :param err:
         :return:
         """
-        log = get_logger(job.jobid)
+        log = get_logger(job.jobid, logger)
+
+        self.pre_setup(job)
+
+        self.post_setup(job)
```

Comment:
This is not correct. Should be called after the setup has completed.

Reply:
updated.